pkg/controllers/cloud/networkpolicy.go

```diff
@@ -262,7 +262,7 @@ func vpcsFromGroupMembers(members []antreanetworking.GroupMember, r *NetworkPoli
 			r.Log.Error(fmt.Errorf(""), "Invalid cloud resource type received", "kind", kind)
 		}
 		if readAnnotations {
-			ownerAnnotations, err := getOwnerAnnotations(e, r)
```

Comment:
Annotation are fields in the object. we should keep it rather than changing it to propeties

Reply:
Changed to properties as we are fetching CloudProvider type too along with annotations from VM crd